scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingMethodImpl.java

```diff
@@ -541,11 +541,9 @@ protected NodeSet getRMNodes(Map<JobId, JobDescriptor> jobMap, int neededResourc
     /**
      * Update all variables for the given scheduled tasks
      */
-    private void updateVariablesForTasksToSchedule(Map<JobId, JobDescriptor> jobMap,
-            LinkedList<EligibleTaskDescriptor> tasksToSchedule) {
+    private void updateVariablesForTasksToSchedule(LinkedList<EligibleTaskDescriptor> tasksToSchedule) {
```

Comment:
You can change this LinkedList into a List type

Reply:
The benefit of using a LinkedList arise when you add or remove from the head of the list which is the case later.